### DISCUSSION TITLE
Increase security by blocking malware and trackers

### DISCUSSION THREAD

**Discussion by theo-bnts** [engageub/InternetIncome#188] [Ideas]
Hi, there !
I'm concerned about the use of my network and found [this article](https://simeononsecurity.com/other/creating-profitable-low-powered-crypto-miners/#increase-security-by-blocking-malware-and-trackers). Maybe it could be interesting to put a simialire protocol in place to block Tor network / torrents. Some states (like France) can sometimes take measures against people who own networks used for such purposes.

**Comment by engageub** [comment]
Hello,
Thank you for sharing the links.

The ports being blocked are assumptions made by the author and are commonly used by applications such as Tor. Users can change these ports before launching the Tor application.

It’s important to note that Tor (The Onion Router) was originally funded and developed by the U.S. government. When a country is aware of Tor usage, they typically block access through the ISP, rather than setting a "bait" to lure users by tracking specific IP addresses and catching them like fish.

Thank you

**Comment by theo-bnts** [comment]
Thank you for your answer and your explanations !
